TombLib/TombLib/LevelData/RoomGeometry.cs

```diff
@@ -43,10 +43,21 @@ public class RoomGeometry
 
         private readonly bool _legacy;
 
-        public RoomGeometry(Room room, bool legacy = false)
+        public RoomGeometry(bool legacy = false)
         {
             _legacy = legacy;
 
+        }
+        public void Build(Room room, bool legacy = false)
+        {
+            VertexPositions.Clear();
+            VertexEditorUVs.Clear();
+            VertexColors.Clear();
+            TriangleTextureAreas.Clear();
+            TriangleSectorInfo.Clear();
+            SharedVertices.Clear();
+            VertexRangeLookup.Clear();
```

Comment:
Could you please benchmark if clearing the collections causes the process to be slower, because that's what I was worrying about when also considering this approach.

Try and create a 100x100 room in TEN and test on that as it takes a very long time to process a room like this.

Reply:
In the case of lists and dictionaries, clearing does not change the capacity. 
This is better than recreating Lists and Dictionaries with a low capacity and then re-filling them; During the refill the collections would need to realloc and rehash their contents. This is not the case when doing `Clear()`.